mod-pubsub-server/src/main/java/org/folio/rest/impl/PubSubImpl.java

```diff
@@ -296,14 +280,6 @@ public void deletePubsubEventTypesSubscribersByEventTypeName(String eventTypeNam
     }
   }
 
-  private MessagingModuleFilter getMessagingModuleFilter(String eventType, ModuleRole role, String tenantId) {
-    MessagingModuleFilter filter = new MessagingModuleFilter();
-    filter.byEventType(eventType);
-    filter.byModuleRole(role);
-    filter.byTenantId(tenantId);
-    return filter;
-  }
-
   private AuditMessageFilter constructAuditMessageFilter(String startDate, String endDate, String eventId, String eventType, String correlationId) {
```

Comment:
Why to use 3 date formats here ? Why not to use SimpleDateFormat.parse() ?

Reply:
We decided to support date, datetime and datetime with timezone formats. To use SimpleDateFormat.parse() we still have to specify the pattern